Makefile

```diff
@@ -80,7 +81,7 @@ defn: $(defn_files)
 
 .build/ocaml/driver-kompiled/interpreter: $(ocaml_files) KRYPTO.ml deps
 	@echo "== kompile: $@"
-	eval $(shell opam config env) \
+	eval $$(opam config env) \
```

Comment:
@msaxena2 what's it about?

Reply:
I approve this. It should not have worked without this fix.